requirements/bench.txt

```diff
@@ -16,4 +16,5 @@ mixsimulator>=0.2.9.5
 hyperopt>=0.2.5
 IOHexperimenter==0.2.8.7
 koncept>=0.2.2
-tensorflow>=2.3.1
\ No newline at end of file
+tensorflow==2.3.1
+tensorflow-estimator==2.3.0
```

Comment:
```suggestion
tensorflow==2.3.1  # temporarily fixing because of errors (see discussion in #940)
tensorflow-estimator==2.3.0  # temporary fix (to be removed?)
```